Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.14 AS builder
+WORKDIR /go/src/github.com/openshift/secondary-scheduler-operator
+COPY . .
+# image-references file is not recognized by OLM
+RUN rm manifests/4*/image-references
+RUN go build -o secondary-scheduler-operator ./cmd/cluster-kube-descheduler-operator
+
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/github.com/openshift/secondary-scheduler-operator/secondary-scheduler-operator /usr/bin/
+# Upstream bundle and index images does not support versioning so
+# we need to copy a specific version under /manifests layout directly
+COPY --from=builder /go/src/github.com/openshift/secondary-scheduler-operator/manifests/4.8/* /manifests/
```

Comment:
I would prefer to go like this,a s recommended by the operator-framwork sdk 
[https://github.com/openshift/cluster-nfd-operator/blob/master/Dockerfile#L12](https://github.com/openshift/cluster-nfd-operator/blob/master/Dockerfile#L12)